### ISSUE TITLE
New release detectron2 v1.0

### ISSUE THREAD

**Issue by bouachalazhar** [facebookresearch/detectron2#5178]
## 🚀 Feature
Since PyTorch v2.0 some functions are not compatible, integer SAM, ViT and GPT. Finally clean up project.

## Motivation & Examples

detectron2 v1.0 can be more stable  and fast if we add `torch.compile()`. It's primordial to create a PyPi project and who can be used by all OS (I think Windows). 

An example for installation detectron2 : 

`python -m pip install detectron2`

**Comment by RiccardoMaistri** [comment]
I agree, with you!. I am just wondering why no more versions has been released since Nov 2021

**Comment by SergiyShebotnov** [comment]
> I agree, with you!. I am just wondering why no more versions of detectron2 have been released since Nov 2021

Everyone probably uses HuggingFace, Meta included.

**Comment by HppZ** [comment]
Sorry, I don't understand, where to get the latest detectron2?
